oat-sa/extension-lti-outcomeui
# feat: new service for formatting results

# Pull request summary

Related to [MS-2021](https://oat-sa.atlassian.net/browse/MS-2021).

We tried to load the preview in manual scoring, the item preview loads the items but the assessment responses cannot be displayed.

A backport will be created for this solution to be integrated in `oat-sa/tao-community:2021.11.16 `

# How to test

1. Install tao instance with `oat-sa/tao-community:2021.11.16`
2. Include the extension `oat-sa/extension-lti-outcomeui:dev-fix/MS-2021/inability-to-preview as 1.2.0`
3. Use [https://ltiapps.net/test/tc.php](https://ltiapps.net/test/tc.php) to see the preview with the answers

# Checklist
